lib/active_model/serializer/associations.rb

```diff
@@ -82,13 +84,15 @@ def associate(reflection)
         end
       end
 
+      # @param [IncludeTree] include_tree (optional)
```

Comment:
if you don't mind, s/optional/defaults to DEFAULT_INCLUDE_TREE when not set/ or something like that?